doc/api/readline.md

```diff
@@ -355,8 +355,8 @@ signal.addEventListener('abort', () => {
 setTimeout(() => ac.abort(), 10000);
 ```
 
-If this method is invoked as it's util.promisify()ed version, it returns a
-Promise that fulfills with the answer. If the question is canceled using
+Use [`util.promisify()`][] to create a version that returns a
+promise that fulfills with the answer. If the question is canceled using
 an `AbortController` it will reject with an `AbortError`.
```

Comment:
```suggestion
Don't use [`util.promisify()`][] to create a version that returns a
promise that fulfills with the answer, use `require('timers/promises').setTimeout` instead. If the question is canceled using
an `AbortController` it will reject with an `AbortError`.
```

Reply:
We can also recommend `AbortSignal.timeout()`

Reply:
Since there's a documented promise API, I think we can just delete all the text, or replace it with something like "See [other API][] for a promise-based version of this API."

Reply:
Yes, removing it could make sense indeed. Worth noting that promise API is still marked as experimental and not available on Node.js <17, so we could expect a large chunk of our users would prefer to rely on `util.promisify`. Folks who want to support older release lines should check the documentation of the older release line.

Reply:
How about we just remove the text entirely?